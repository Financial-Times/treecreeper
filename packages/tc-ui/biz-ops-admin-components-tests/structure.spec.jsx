const React = require('react');

const {
	SectionHeader,
} = require('../../../src/templates/components/structure.jsx');

const { render } = require('../../testHelpers/component');

describe('<SectionHeader />', () => {
	it('Display of title and edit button', () => {
		const component = render(
			<SectionHeader title="A Nice Title Wth Edit" code="code" />,
		);
		expect(component).toMatchSnapshot();
	});

	it('Display of just title', () => {
		const component = render(<SectionHeader title="Just A Nice Title" />);
		expect(component).toMatchSnapshot();
	});

	it('Display with no details', () => {
		const component = render(<SectionHeader />);
		expect(component).toMatchSnapshot();
	});
});
