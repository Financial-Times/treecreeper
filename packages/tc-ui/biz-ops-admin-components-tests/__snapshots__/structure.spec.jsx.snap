// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionHeader /> Display of just title 1`] = `<h2 id="just-a-nice-title" class="section-heading">Just A Nice Title</h2>`;

exports[`<SectionHeader /> Display of title and edit button 1`] = `<h2 id="a-nice-title-wth-edit" class="section-heading">A Nice Title Wth Edit</h2>`;

exports[`<SectionHeader /> Display with no details 1`] = `<h2>no title provided</h2>`;
